ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -204,6 +204,25 @@ conformance_test(
     ],
 )
 
+conformance_test(
+    name = "conformance-test-multi-participant-command-deduplication",
```

Comment:
Does it belong to this PR?

Reply:
This is basically the validation that the changes work, it would make no sense to merge without it as we can't confirm the changes are valid.